core/src/avm1/object/script_object.rs

```diff
@@ -53,11 +53,11 @@ impl<'gc> Watcher<'gc> {
 
 #[derive(Copy, Clone, Collect)]
 #[collect(no_drop)]
-pub struct Object<'gc>(GcCell<'gc, ObjectData<'gc>>);
+pub struct Object<'gc>(Gc<'gc, RefLock<ObjectData<'gc>>>);
```

Comment:
That doesn't change anything semantically, right? `GcCell` is a `Gc<RefLock>`

Wouldn't it be better to leave `GcCell` for now and revisit when we can get rid of locking the whole object?